spec/money_spec.rb

```diff
@@ -175,6 +175,11 @@
     Money.new(16.51).round(1).should == Money.new(16.5)
   end
 
+  it "can be converted form an amount of cents that cannot be represented as an IEEE 754 double precision float" do
+    amount = 2**53 + 1
+    expect(Money.from_cents(amount).cents).to eq(amount)
```

Comment:
Because many methods use `@value`, the cents are good, but the `.to_s`, `to_json`, `+`, etc, will produce bad results